atlasdb-workload-server-distribution/src/main/java/com/palantir/atlasdb/workload/WorkloadServerLauncher.java

```diff
@@ -270,6 +275,27 @@ private WorkflowAndInvariants<Workflow> createSingleRowTwoCellsWorkflowValidator
                 .build();
     }
 
+    private WorkflowAndInvariants<Workflow> createBankBalanceWorkflow(
+            AtlasDbTransactionStoreFactory transactionStoreFactory,
+            BankBalanceWorkflowConfiguration workflowConfig,
+            LifecycleEnvironment lifecycle) {
+        ExecutorService executorService = lifecycle
+                .executorService(BankBalanceWorkflows.class.getSimpleName())
+                .maxThreads(workflowConfig.maxThreadCount())
```

Comment:
We also need `.minThreads(workflowConfig.maxThreadCount())` if we actually want to run this concurrently.